torchvision/prototype/transforms/functional/__init__.py

```diff
@@ -68,6 +68,7 @@
     five_crop,
     five_crop_image_pil,
     five_crop_image_tensor,
+    hflip,
```

Comment:
Question: Do we want to define all aliases in one single place? Perhaps all aliases should leave here instead on the submodules. No strong opinion. Thoughts @pmeier @vfdev-5 ?

Reply:
I think they can live in appropriate submodules

Reply:
OK keeping as is then